examples/add/webpack.config.js

```diff
@@ -14,12 +14,6 @@ module.exports = {
         new WasmPackPlugin({
             crateDirectory: path.resolve(__dirname, ".")
         }),
-        // Have this example work in Edge which doesn't ship `TextEncoder` or
-        // `TextDecoder` at this time.
-        new webpack.ProvidePlugin({
-          TextDecoder: ['text-encoding', 'TextDecoder'],
-          TextEncoder: ['text-encoding', 'TextEncoder']
-        })
     ],
     mode: 'development',
```

Comment:
Maybe it would be faster in `release` mode?

Reply:
It's called `production` and it's [about the same](https://github.com/Systemcluster/wasm-bindgen/actions/runs/10273924466/usage). Both `test_examples` as well as `build_examples` stays the same within fluctuations.